api/routers/groups.py

```
@@ -0,0 +1,230 @@
+        select(HostGroup)
+        .options(selectinload(HostGroup.hosts), selectinload(HostGroup.users))
+        .order_by(HostGroup.name)
+    )
+    return [group_out(g) for g in result.scalars().all()]
+
+
+@router.post("", status_code=201)
+async def create_group(
+    request: Request,
+    body: CreateGroupIn,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)